app/pb/constant_data.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package org.couchers.api.constant_data;
+
+import "google/protobuf/empty.proto";
+
+service ConstantData {
+  // This is a secure service: a user needs to be authenticated and not jailed to call functions here, refer to auth.proto and jail.proto
```

Comment:
I think the service should not exist. But if it does, it should be open, to permit a frontend to present a list of valid choices in the user registration form.